.bazelci/presubmit.yml

```diff
@@ -202,6 +209,19 @@ tasks:
     build_targets: *default_linux_targets
     test_targets: *default_linux_targets
     build_flags: *aspects_flags
+  ubuntu2004_min_rust_version:
+    name: "Min Rust Version"
+    platform: ubuntu2004
+    shell_commands: *min_rust_version_shell_commands
+    build_targets: *single_rust_channel_targets
+    test_targets: *single_rust_channel_targets
```

Comment:
While I think we should be testing the rules all together, there might be value in having more granular testing for this since the current failure on `1.58.0` is a failure in a transitive crate_universe dependency. Open to suggestions on how this should be structured.

Reply:
@scentini @krasimirgg any thoughts here?